tasks/Bullard Houses.md

```diff
@@ -0,0 +1,48 @@
+# Summary
+The Bullard House experiment is designed to test negotiations and agreement-bias.  In this experiment the two roles are designed to be at odds with their interests, which creates a negative bargaining zone.
+
+# References
+Cohen, T., G. Leonardelli, and L. Thompson. "The agreement bias in negotiation: Teams facilitate impasse." 23rd annual conference of the International Association for Conflict Management (IACM), Boston, US. 2010.
+
+Link: https://d1wqtxts1xzle7.cloudfront.net/51359748/The_Agreement_Bias_in_Negotiation_Teams_20170115-19001-1910usu-with-cover-page-v2.pdf?Expires=1630165799&Signature=Leq9jWOmOoglLUt7pY1HrlvFlv47s314LbCYqHmpGIlhMLYkvCmDk5O8qIhvTw8LR75oyeaMHvX34frVvw6KbYkMGLaBaIj3hawbz-VMEwtskNCS-5YwB30ge9Dte~PYf20bSmoLeXAa2uIw1rhsrdO7FOLU9dh0yVbXlpnYpHB6T4o-zWFiZJompKkdHgIjjC6pawHzMms9DFBmPxZJwgaAvEuUAbKgE3uZBUGpJCeM-is8TKiiX-91ipoWwosfZxArMaP2AM0UwGmWiiyEH7JwCgF1YrByhGQdCFhObUm6wGPabxyBvYfvZc2H9F~vDwPpZcFMPSS9ALOuxpHDuA__&Key-Pair-Id=APKAJLOHF5GGSLRBV4ZA
+
+
+# Stimuli
+## The visual components
+N/A
+
+
+## Materials for alternative versions of the experiment 
+Alternate versions of the experiment allow you to introduce teams.  These experiments allow you to test the effects of group negotiations.  Different formats tested in, The agreement bias in negotiation: Teams facilitate impasse were, one-on-one, two-on-two and three-on-three.
+
+
+# Procedure
+Groups of participants were selected from MBA students.  (At this point you can break the groups into teams, or use one-on-one negotiations. Teams are assigned randomly.)  Participants are then given their role materials one week prior to negotiations.
+
```

Comment:
Participants are assigned to negotiate in one of three formats:
>  one-on-one 
> two-on-two 
> three-on-three We
All parties are assigned a counterpart of the same team size as their own